tool/npm_template/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "gltf-validator",
+    "description": "Library for validating glTF 2.0 assets, compiled to JS.",
+    "license": "Apache-2.0",
+    "bugs": "https://github.com/KhronosGroup/glTF-Validator/issues",
+    "homepage": "https://github.com/KhronosGroup/glTF-Validator",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/KhronosGroup/glTF-Validator"
+    },
+    "author": {
+        "name": "The Khronos Group Inc."
+    },
+    "main": "index.js",
+    "keywords": [
+        "gltf",
+        "webgl",
+        "3d"
+    ]
```

Comment:
What happened to the version number?  I think a version number is required here, no?

Reply:
Version number will be present in `build/npm/package.json`. It comes from `pubspec.yaml` on compilation.